src/test/bundle_test.ts

```diff
@@ -35,7 +37,7 @@ class FileTransform extends Transform {
     this.transform = transform;
   }
   _transform(f: File, _encoding: string, cb: () => void) {
-    this.transform(this, f);
+    this.transform(this, f.clone());
```

Comment:
The main reason this clone was necessary is that tests using this transform were trying to demonstrate that the BuildBundler was processing value from the incoming stream, but without cloning the file in the transform here in the test, the test would pass if loader read the old file since the contents were changed in the file.  By forcing the clone, I demonstrate that the BuildBundler is processing the file that's coming in on the stream.  There's nothing _AFAIK_ that would break if the `clone()` was removed.  Additionally, it should be established best practice for transform streams to produce new objects instead of mutating existing ones if they are going to support parallelization etc, so I don't think this means the underlying behavior expected is any different.